AddRoutineView.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  AddRoutineView.swift
+//  OnTrack
+//
+//  Allows user to add a new routine to their list of routines.
+//
+//  Created by Noah Sadir on 4/4/24.
+//
+//  Contributors: David
+//
+
+import SwiftUI
+
+struct AddRoutineView: View {
+    @State var id: String = ""
+    @State var title: String = ""
+    @State var description: String = ""
+    @State var category: String = ""
+    @State var frequency: Int = 0
+    @State var dates: String = ""
+    @State var streakLength: Int = 0
+    
+    @StateObject var vm = ViewModel()
+    
+    var body: some View {
+        VStack {
+            HStack {
+                Text("OnTrack")
+                    .font(.largeTitle)
+                    .bold()
+                    .padding(.leading)
+                
+                Spacer()
+                
+                
+            }
+            
+            Text("Add Routine")
+                .padding(.vertical)
+                .font(.largeTitle)
+               .italic()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
```

Comment:
Generally avoid manually setting RGB values. Swift has some built in colors which you can use (e.g. `.green`)